embedding-reports/display-reports-in-applications/web-application/native-blazor-report-viewer/how-to-use-native-blazor-report-viewer.md

```diff
@@ -1,40 +1,40 @@
 ---
 title: How to Use Native Blazor Report Viewer
-page_title: How to Use Native Blazor Report Viewer 
-description: How to Use Native Blazor Report Viewer
+page_title: Integrating Native Blazor Report Viewer in Blazor Server or WebAssembly Application
+description: "Learn how to configure and integrate the Native Blazor Report Viewer built on top of Telerik Blazor UI widgets in you Blazor Server or WebAssembly Application."
```

Comment:
I think the correct term is Telerik UI for Blazor https://www.telerik.com/blazor-ui